### PR TITLE
Update WASM 0.2.0, Refactor and Zmenu fix

### PR DESCRIPTION
**Pull request by msmanoj**
### Description
- Updated WASM to 0.2.0
- Separated the functions within the main class file into separate files

I have also pulled in all the changes from https://github.com/Ensembl/ensembl-genome-browser/pull/2

### CODE REVIEW

File: src/methods/handleIncoming.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        
+        [...subscriptionsToAction.values()].forEach( subscription => {
+        
+        const formattedIncoming = formatIncoming(type, payload);
+        if(!formattedIncoming){
+            console.error(payload);
+            return;
+        }
+        subscription(formattedIncoming);
+        })
```
Comment (azangru) [inline src/methods/handleIncoming.ts]: something strange with indentation inside this forEach

File: src/ensemblGenomeBrowser.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import { 
+    ConfigData,
+    GenomeBrowserType,
+    IncomingActionType,
+    OutgoingAction,
+    Subscribe,
+    SubscribeArgs
+  } from './types';
+  
+  import send from './methods/send';
```
Comment (azangru) [inline src/ensemblGenomeBrowser.ts]: Starting from this import and down, everything is indented two spaces to the right.

File: src/methods/subscribe.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    }
+    
+    const unsubscribe = () => {
+        subscriptions.get(actionType)?.delete(subscriber);
+    };
+    
+    return { 
+        unsubscribe
+    }
+};
```
Comment (azangru) [inline src/methods/subscribe.ts:23]: This file uses 4-space indents 🤷‍♂️

**Comment by msmanoj** [comment]
I merged this PR accidentally from the command line and I am trying to bring back this PR.

**Pull request merged**